docs/guides/connect-aepp-to-wallet.md

```diff
@@ -59,12 +59,18 @@ async created () {
 ```js
 methods: {
   async scanForWallets()  {
+    let connectedCallback
```

Comment:
Well, I was not able to make this work.
I think It will require some more changes when using this solution.

What I did here, was that I found out that the provided example is not working correctly
https://aeternity.com/aepp-sdk-js/v11.0.0/guides/connect-aepp-to-wallet/

I got the advice from @davidyuk on this solution. 
Tested it and it worked. If you want to discuss what is the clearest solution to this, better consult with David - as this is your domain, guys. I don't have a strong opinion on this.

Just wanted to report the bug and document a possible solution.